metrics-operator/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-operator
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+  {{- include "chart.labels" . | nindent 4 }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metrics-operator
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/part-of: keptn-lifecycle-toolkit
```

Comment:
please add more descriptive k8s default labels :)

Reply:
I tought we agreed to leave this as part of klt... or?

Reply:
aha ok, but then please add more of the default labels :)